sklearn/cluster/optics_.py

```diff
@@ -425,6 +422,42 @@ def fit(self, X, y=None):
 
     # OPTICS helper functions
 
+    def _calculate_core_distances_(self, X, nbrs, working_memory=None):
```

Comment:
As a non-native English speaker, I wonder if "calculate" to is natural or not. It sounds French to my ears and I would have thought that "compute" would be more idiomatic but I would love to get the feedback of a native English speaker.

Any way, it's a private function so it's not necessary to change this.